app/assets/stylesheets/_grid.scss

```diff
@@ -11,6 +11,21 @@
   &.center {
     @include justify-content(center);
   }
+
+  .form-field {
+    label {
+      line-height: 40px;
+    }
+
+  .Select-control {
+      padding-bottom: 6px;
+      height: 34px;
+    }
+
+    input:not(.input-small) {
+      margin-top: 5px;
+    }
+  }
```

Comment:
These style don't _declare the grid_ but how to style form inputs when they are _within a grid_, so I think they'd fit better in a stylesheet related to forms.

Maybe `_forms.scss`? But `.Select-control` is defined in `_react_select.scss`, so I'm not sure. I'd love your input @devduarte @straight-shoota!

Reply:
I moved them to `_forms.scss` and let the `.Select-control` specific style for forms in there since they have to be form-fields **and** Select-control. If you think that is not ok I change it to` _react_select.scss`.